playbooks/roles/docker/files/datetime.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=starts datetime container
+After=docker.service
+Requires=docker.service
+
+[Service]
+User=wannabe
+TimeoutStartSec=0
+Restart=always
+ExecStartPre=-/usr/bin/docker stop datetime
+ExecStartPre=-/usr/bin/docker rm datetime
+ExecStart=/usr/bin/docker run -d --name datetime datetime:habuild
+ExecStopPre=-/usr/bin/docker stop -t 11 datetime
```

Comment:
ExecStopPre causes error when running this service -- ExecStop (without the "Pre") runs OK